src/state/auth/actions.js

```diff
@@ -3,11 +3,20 @@
 import superagent from 'superagent';
 import cookie from 'react-cookies';
 
+const bearerToken = () => {
+  return cookie.load('auth');
+};
+
```

Comment:
this seems like an unnecessary  wrapping of a function in a function. I think just calling cookie.load('auth') where you need it is fine, but I guess it's slightly more symatic